docs/ovs-pipeline.md

```diff
@@ -376,28 +377,41 @@ In the future this table may support an additional mode of operations, in which
 it will implement kube-proxy functionality and take care of performing
 load-balancing / DNAT on traffic destined to services.
 
-### CnpEgressRuleTable (45)
+### CnpEgressRuleTables (45-49)
```

Comment:
The document will change again if https://github.com/vmware-tanzu/antrea/pull/1066 is merged? Assuming you are fine with updating the doc again at that point, I am fine with merging this now :)

Reply:
yeah.. however it is unlikely to be merged before tomorrow? and for 0.10.0 if Tier CRDs are accepted, then we may update #1066 and doc together.